templates/colors/swift2.stencil

```diff
@@ -35,12 +35,12 @@ struct {{enumName}} {
 {{sp}}  {% for color in colors %}
 {{sp}}  /// <span style="display:block;width:3em;height:2em;border:1px solid black;background:#{{color.red}}{{color.green}}{{color.blue}}"></span>
 {{sp}}  /// Alpha: {{color.alpha|hexToInt|int255toFloat|percent}} <br/> (0x{{color.red}}{{color.green}}{{color.blue}}{{color.alpha}})
-{{sp}}  static let {{color.name|swiftIdentifier|escapeReservedKeywords}} = {{enumName}}(rgbaValue: {% call rgbaValue color %})
+{{sp}}  static let {{color.name|swiftIdentifier:"pretty"|escapeReservedKeywords}} = {{enumName}}(rgbaValue: {% call rgbaValue color %})
 {{sp}}  {% endfor %}
 {% endmacro %}
   {% if palettes.count > 1 %}
   {% for palette in palettes %}
-  enum {{palette.name|swiftIdentifier|titlecase|snakeToCamelCase|escapeReservedKeywords}} {
+  enum {{palette.name|swiftIdentifier:"pretty"|escapeReservedKeywords}} {
```

Comment:
Which is also a reason why I don't like `swiftIdentifier` to uppercase the first letter 😉
e.g. `iOSModel` --[swiftIdentifier]--> `IOSModel` --[lowerFirstWord]--> `iosModel` 😉

But that's a different debate I guess ^^